src/bootstrap/index.js

```diff
@@ -9,6 +9,12 @@ import { createRm } from './rm.js'
  */
 export function createBootstrap (config) {
   return {
+    /**
+     * TODO: Add bootstrap.add.default
+     * TODO: Remove bootstrap.clear
+     * TODO: Remove bootstrap.reset
+     * TODO: Add bootstrap.rmAll
```

Comment:
As a user and maintainer, I think that would be a good move forward, but isn't the current pattern for other APIs that I've seen in the RPC client, so I think if we change it here, we should definitely change it everywhere. 

As a counter-point, there is some benefit to separating the API so accidents are more challenging, but I that's a stretch at best.

Reply:
https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-bootstrap-rm-all lists the command on it's own line, so my thinking with the TODO was that it would make sense as it's own command (seems to be existing pattern)

Reply:
opened https://github.com/ipfs/kubo/issues/9509 for this